src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3099,6 +3100,7 @@ def transform(dataset):
         Bool('force_size'),
         Inheritable(Str('comments')),
         Inheritable(Str('sync', enum=['STANDARD', 'ALWAYS', 'DISABLED'])),
+        Inheritable(Str('snapdev', enum=['HIDDEN', 'VISIBLE']), has_default=False),
```

Comment:
I wonder if we should default to `VISIBLE` on new zvol creation. If we do this though we may want to try creating 10K snapshots on a zvol and see whether anything breaks.